hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/DefaultRecordStore.java

```diff
@@ -662,27 +662,27 @@ public Data readBackupData(Data key) {
     }
 
     @Override
-    public MapEntrySet getAll(Set<Data> keys) {
+    public MapEntries getAll(Set<Data> keys) {
         checkIfLoaded();
         final long now = getNow();
 
-        final MapEntrySet mapEntrySet = new MapEntrySet();
```

Comment:
just a detail: the naming is a bit confusing. you have "entries" and "loadedEntries" in the scope of this method.